components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/internal/FeatureManifestInterface.kt

```diff
@@ -43,4 +43,8 @@ interface FeatureManifestInterface<T> {
     fun getFeature(featureId: String): FeatureHolder<*>?
 
     fun getCoenrollingFeatureIds(): List<String>
+
+    fun prefsToBranchesMap(): Map<String, String>
+
+    fun prefsToFeatureIdsMap(): Map<String, String>
```

Comment:
We need to be able to map from `(featureId, variable)` to `(pref, branch)`. I think we want `Map<String, Map<String, String>>` here.